service/history/replicatorQueueProcessor.go

```diff
@@ -56,15 +56,22 @@ type (
 )
 
 var (
-	errUnknownReplicationTask = errors.New("Unknown replication task")
-	errHistoryNotFoundTask    = errors.New("History not found")
+	errUnknownReplicationTask = errors.New("unknown replication task")
```

Comment:
Can we rename this -- replicatorQueueProcessor. I don't think there is a processor anymore after your change. Having both ReplicatorQueueProcessor and ReplicationTaskProcess is very confusing.

Reply:
One of them will be removed eventually, hopefully in the +1 release. So I don't want to put too much work naming these.